rfcs/0166-nix-formatting.md

```diff
@@ -0,0 +1,1063 @@
+---
+feature: nix_formatting
+start-date: 2021-08-17
+author: piegamesde
+co-authors: infinisil
+pre-RFC reviewers: tomberek, 0x4A6F
+shepherd-team:
```

Comment:
This is more a formality than anything else, but I nominate @tomberek @0x4A6F to continue shepherding this (and also @infinisil, if that's allowed as co-author)

Reply:
The RFC Steering Committee discussed this and we think it is fine to have a co-author on the shepherd team. However in this case it would be good to try to get an extra shepherd over the minimum to ensure diversity of opinions on the team. However we do no think that is strictly required.

Reply:
I also accept the nomination!

Reply:
accept shepherd nomination

Reply:
I'd like to nominate myself as a shepherd :)

Reply:
See https://github.com/NixOS/rfcs/pull/166#discussion_r1426605181 :)